feature-libs/asm/components/customer-selection/customer-selection.component.ts

```diff
@@ -89,13 +89,13 @@ export class CustomerSelectionComponent implements OnInit, OnDestroy {
 
   onSubmit(): void {
     if (this.customerSelectionForm.valid && Boolean(this.selectedCustomer)) {
-      this.submitEvent.emit({ customerId: this.selectedCustomer.customerId });
+      this.submitEvent.emit({ customerId: this.selectedCustomer?.customerId });
```

Comment:
Same as above, the `?` should not be needed, would also consider dropping `Boolean` in the line before.

Reply:
Same solution as above in the same file.  The compiler does not understand the check is done with `Boolean(this.selectedCustomer)` but does understands it with `!!this.selectedCustomer`.

I was then able to remove the `?`